client/my-sites/migrate/section-migrate.jsx

```diff
@@ -76,11 +87,35 @@ class SectionMigrate extends Component {
 			 * Note this migrationStatus is local, thus the setState vs setMigrationState.
 			 * Call to updateFromAPI will update both local and non-local state.
 			 */
-			this.setState( { migrationStatus: 'inactive', errorMessage: '' }, this.updateFromAPI );
+			this.setState(
+				{
+					migrationStatus: 'inactive',
+					errorMessage: '',
+				},
+				this.updateFromAPI
+			);
 		} );
 	};
 
 	setMigrationState = state => {
```

Comment:
Is this really updating the migration state? Or is it updating the migration status?

Reply:
It's tricky finding a good name for this method: it's resetting the component's internal state, and – if the update includes `migrationStatus` – also the application state.